src/silverlabnwb/nwb_file.py

```diff
@@ -190,8 +190,11 @@ def add_core_metadata(self):
         self._write()
 
     def add_subject(self, subject_data):
-        """Add the subject information from YAML config to the NWB file."""
-        subject = Subject(**subject_data)
+        """Add the valid subject information from YAML config to the NWB file.
+
+        Use pynwb default if a subject information entry is None."""
```

Comment:
```suggestion
        Use pynwb default if a subject information entry is None.
        """
```
per [convention](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings) (and similarly below).